haystack/nodes/other/route_documents.py

```diff
@@ -41,6 +41,8 @@ def __init__(self, split_by: str = "content_type", metadata_values: Optional[Lis
         # If we split list of Documents by a metadata field, number of outgoing edges might change
         if split_by != "content_type" and metadata_values is not None:
             self.outgoing_edges = len(metadata_values)
+        else:
+            self.outgoing_edges = 2
```

Comment:
Soon `content_type` values might increase in number. We will have `text`, `table`, `audio`, `image`, and who knows how many more. Do you think there is a way to make this number actually reflect the number of content types existing in Haystack? Maybe we can store them in a constant in `haystack/schema.py` and read its length?